src/bridge.ts

```diff
@@ -110,6 +113,8 @@ export interface BridgeController {
             PossiblePromise<ThirdpartyUserResponse[]>;
         parseUser?(userid: string): PossiblePromise<ThirdpartyLocationResponse[]>;
     };
+
+    userActivityTracker?: UserActivityTracker;
```

Comment:
Interesting placement choice, we tend to leave the controller for callbacks only. Was there a reason you wanted it here rather than the config?

Reply:
As far as the bridge (the MAB) is concerned, it is basically a callback (it doesn't utilize anything else, for now at least) – controller felt like a fitting place for something that *does* things. This used to have a different API where you passed UAT configuration in opts, but that violated DI and was silly in all sorts of ways.

Reply:
Seems reasonable. At some point we might want to revisit our bridge configuration as a whole, as it feels like this might apply to other things.